doc/source/installation.rst

```diff
@@ -32,11 +32,22 @@ bundeled for syslog and journald logging as well. These could be used similarly
 
 3. Then install node agent itself::
 
+For installing node agent following dependencies have to be manually installed:
+
+    $ yum install libffi-devel gcc python-devel openssl-devel
+
+Clone and install the node-agent
+
     $ git clone https://github.com/Tendrl/node_agent.git
     $ cd node_agent
     $ workon node_agent
     $ pip install .
 
+One of the dependency of node-agent needs a higher version of setuptools, So user
+has to manually upgrade the setuptools package to alteast setuptools>=11.3
```

Comment:
yes, this latest version of setuptools is needed only for node_agent. One of the dependency of ansible has this dependency.